src/components/AddPoll/addPoll.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { InputGroup, FormControl, Button, Modal } from "react-bootstrap";
+import { PlusCircleFill } from "react-bootstrap-icons";
```

Comment:
Remove this File and related function from this branch @PuniaMohit 
If its a login Signup feature branch, code should be related to only this